addons/account/views/account_reconcile_model_views.xml

```diff
@@ -206,6 +213,41 @@
                                         </tree>
                                     </field>
                                 </group>
+                                <!-- For writeoff button of counterpart type Invoice or Bill-->
```

Comment:
why duplicating the groups and only dislpaying some of the columns ? Also, why enforce the percentage of statement line for the `amount_type` field ? There is nothing in the spec about this :thinking: I know it might make the task harder, but we might want to keep for example the 'fixed' and 'from label' choices as well, not sure about the 'percentage of balance' though. I just discussed with CHKL and indeed, he would like to keep the original group, not duplicating it. But I guess there is a valid reason why you did like this so don't hesitate to provide explanations. 

Also, I believe there is currently an issue with reco model when you use multiple lines (simply for journal entries then). For example, for a transaction of 100, and a reco model with 2 lines using percentage of statement line, one at 80% and the other at 20%, it will create a line at 80 and the second at 4, which is quite unexpected I suppose. So we might want to fix this issue first, then use somehow the same system.